### DISCUSSION TITLE
Adding Accounts using csv file, unable to logon to them

### DISCUSSION THREAD

**Discussion by Gordo-99** [rejetto/hfs#769] [Q&A]
I have used HFS for years and want to move to version 3.  I have about 175 accounts so I was trying to import them using the csv option and when I do the accounts cannot logon.  All of the accounts show up, they even have the proper redirect but I cannot logon even if I set the password.  The csv file comes out of Excel, looks proper, I open it in notepad and there is nothing weird in it, no extra characters or spaces but I did notice the case on the account name gets changed so I tried with upper and lower case.  For example if I import a simple csv file with format as below, the accounts show up as me1 & me2, they have the folder redirect ME1/ & ME2/ but I cannot logon to them even after changing the password.  Am I doing something wrong or is this a bug
ME1,PasSWord1#,,ME1/
ME2,PasSWord2#,,ME2/

**Comment by rejetto** [comment]
hey!
this is a bug, and thanks for reporting.
I'm working on it right now.

**Comment by rejetto** [comment]
ok, i've fixed the problem.
It was NOT the import, but the login, so you don't have to reimport.
Can you install this version and confirm that it works for you?
Go to admin-panel / home, and right-click on check-for-updates
https://github.com/user-attachments/files/17305826/hfs-windows-x64-0.54.0-beta5.6.zip

**Comment by rejetto** [comment]
in the meantime i published 0.54.0-beta6
